### PR TITLE
Add info about changelog tool to contributor guide

### PR DESCRIPTION
**Pull request by jonathansadowski**
### All Submissions:

-   [ ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR adds info about the creation of change files to the contributor guide. I added this, as that section was removed from the PR template in #37555

Closes # .

### How to test the changes in this Pull Request:

- [ ] Have you followed the [Writing high-quality testing instructions guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions)?

1. Verify that the changes to the contributor guide are correct and understandable.
2.
3.

### Other information:

-   [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [ ] Have you created a changelog file for each project being changed, ie `pnpm --filter=<project> changelog add`?
-   [ ] Have you included testing instructions?

### FOR PR REVIEWER ONLY:

-   [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -39,7 +39,8 @@ If you have questions about the process to contribute code or want to discuss de
 - Ensure that your code supports the minimum supported versions of PHP and WordPress; this is shown at the top of the `readme.txt` file.
 - Push the changes to your fork and submit a pull request on the trunk branch of the WooCommerce repository.
 - Make sure to write good and detailed commit messages (see [this post](https://chris.beams.io/posts/git-commit/) for more on this) and follow all the applicable sections of the pull request template.
-- Please avoid modifying the changelog directly or updating the .pot files. These will be updated by the WooCommerce team.
+- Please create a change file for your changes by running `pnpm --filter=<project> changelog add`.
```
Comment (lsinger) [inline .github/CONTRIBUTING.md]:
```suggestion
- Please create a change file for your changes by running `pnpm --filter=<project> changelog add`. For example, a change file for the WooCommerce Core project would be added by running `pnpm --filter=woocommerce changelog add`. 
```

**Review by lsinger — DISMISSED** [review]
Looks good, but let's make it easier to understand the format of the `<project>` string by giving the probably most commonly used project name as an example. 👍

**Comment by jonathansadowski** [comment]
> Looks good, but let's make it easier to understand the format of the `<project>` string by giving the probably most commonly used project name as an example

Great suggestion. Updated w/ the suggestion.

**Review by lsinger — APPROVED** [review]
LGTM! 🚢

**Pull request merged**